packages/client/src/index.ts

```diff
@@ -28,9 +27,10 @@ export function getSignature(
 ): DecryptedMessage | null {
```

Comment:
Returning an optional type seems bad for developers using this library.
Like utils, it seems that the optional type should be used only inside the library.